build.sbt

```diff
@@ -242,6 +242,9 @@ lazy val commonSettings =
         "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonV,
         "com.fasterxml.jackson.core" % "jackson-core" % jacksonV,
         "com.fasterxml.jackson.core" % "jackson-databind" % jacksonV,
+
+        "io.dropwizard.metrics5" % "metrics-core" % dropWizardV,
```

Comment:
why do we use 5.0.0-rc3 exactly ? there's 5.0.0-rc11 already available ? https://search.maven.org/artifact/io.dropwizard.metrics5/metrics-core/5.0.0-rc11/bundle

Reply:
good point, I'll upgrade...